### PR TITLE
chore: add README & LICENSE

### PR DESCRIPTION
**Pull request by matmalkowski**
As per title, adding initial README file & LICENSE. It will need some work for sure, I've left the missing links as TODO

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020-present Burkhard Reffeling, Maciej Malkowski, McKayla Washburn
```
Comment (victorhqc) [inline LICENSE.md:3]: Should we change this to "contributors" or something more generic? Or are we gonna update this everytime there's a new contributor? If so, then Roman and myself are omitted :(

Comment (matmalkowski) [reply LICENSE.md:3]: @victorhqc I think copyrights cannot be owned by something generic like contributors - maybe we should use Klarna legal entity as the copyright holder?

Comment (victorhqc) [reply LICENSE.md:3]: Why not as "Github contributors" or "Repository contributors".

Comment (victorhqc) [reply LICENSE.md:3]: https://github.com/klarna/electron-redux/graphs/contributors

Comment (matmalkowski) [reply LICENSE.md:3]: Cause contributors have to be defined by name right? We cannot have implicit list in a license. Usually you would have a CLA signed before contributing so we don't append every single person to that list. Maybe lets add the ones that we think should be there, like you mentioned you and Roman? @victorhqc

File: README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+# with npm
+npm install electron-redux@alpha
+```
+
+For more details, please see [the Installation docs page](#todo).
+
+## Documentation
+
+electron-redux docs are located at **electron-redux.js.org**. You can find there:
```
Comment (hardchor) [inline README.md:30]: Can we discuss whether we can perhaps get a klarna.com URL?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+-   [Getting started](#todo)
+-   [Recipes](#todo)
+-   [API Reference](#todo)
+
+## Quick start
+
+electron-redux comes as a [Redux StoreEnhancer](https://redux.js.org/understanding/thinking-in-redux/glossary#store-enhancer). To initialize your stores using it, you just need decorate your stores in the `main` and `renderer` process of electron with provided enhancers:
```
Comment (hardchor) [inline README.md:38]:
```suggestion
electron-redux comes as a [Redux StoreEnhancer](https://redux.js.org/understanding/thinking-in-redux/glossary#store-enhancer). To initialize your stores, you just need to decorate them in the `main` and `renderer` processes of electron with their respective enhancers:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+```ts
+// renderer.ts
+import { rendererStateSyncEnhancer } from 'electron-redux'
+
+const store = createStore(reducer, rendererStateSyncEnhancer())
+```
+
+That's it!
+
+You are now ready to fire actions in any of the stores / processes, and depending on the [scope](#scoped-actions) the main store will broadcast them to correct processes.
```
Comment (hardchor) [inline README.md:56]:
```suggestion
You are now ready to fire actions from any of your processes, and depending on the [scope](#scoped-actions) the main store will broadcast them across your application.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+Please check out the [docs](#todo) for more recipes and examples!
+
+## Actions
+
+Actions fired **MUST be [FSA](https://github.com/acdlite/flux-standard-action#example)-compliant**, i.e. have a type and payload property. Any actions not passing this test will be ignored and simply passed through to the next middleware.
```
Comment (hardchor) [inline README.md:62]:
```suggestion
Actions fired **MUST be [FSA](https://github.com/acdlite/flux-standard-action#example)-compliant**, i.e. have a `type` and `payload` property. Any actions not passing this test will be ignored and simply passed through to the next middleware.
```

Comment (hardchor) [reply README.md:62]: > to the next middleware

Is this still correct since we're using an enhancer instead of middleware?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+> Nota bene, `redux-thunk` is not FSA-compliant out of the box, but can still produce compatible actions once the async action fires.
+
+Furthermore, actions (and that includes payloads) **MUST be serializable**.
+
+> You can extend default JSON serialization used, by [providing your own serialization/deserialization functions](#todo).
+
+### Scoped actions
+
+By default, all actions are broadcasted to all registered stores. However, some state should only live in the renderer (e.g. isPanelOpen). electron-redux introduces the concept of action scopes.
```
Comment (hardchor) [inline README.md:72]:
```suggestion
By default, all actions are broadcast to all registered stores. However, some state should only live in the renderer (e.g. `isPanelOpen`). electron-redux introduces the concept of action scopes.
```

**Review by victorhqc — APPROVED** [review]

**Pull request merged**